backend/src/test/java/com/festago/bookmark/application/BookmarkAppendValidatorTest.java

```diff
@@ -0,0 +1,52 @@
+package com.festago.bookmark.application;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import com.festago.bookmark.domain.Bookmark;
+import com.festago.bookmark.domain.BookmarkType;
+import com.festago.bookmark.repository.MemoryBookmarkRepository;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayNameGeneration;
+import org.junit.jupiter.api.DisplayNameGenerator.ReplaceUnderscores;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.EnumSource;
+
+@DisplayNameGeneration(ReplaceUnderscores.class)
+@SuppressWarnings("NonAsciiCharacters")
+class BookmarkAppendValidatorTest {
+
+    MemoryBookmarkRepository bookmarkRepository = new MemoryBookmarkRepository();
+    BookmarkAppendValidator bookmarkAppendValidator = new BookmarkAppendValidator(bookmarkRepository);
+
+    @BeforeEach
+    void setUp() {
+        bookmarkRepository.clear();
+    }
+
+    @ParameterizedTest
+    @EnumSource(value = BookmarkType.class)
+    void 같은_타입에_북마크_갯수가_이미_12개_이상이면_예외(BookmarkType bookmarkType) {
+        // given
+        Long memberId = 1L;
+        for (int i = 0; i < 12; i++) {
+            bookmarkRepository.save(new Bookmark(bookmarkType, (long) i + 1, memberId));
+        }
```

Comment:
크게 중요한건 아니지만.. for문의 변수 타입을 long으로 하면 형변환이 필요 없을 것 같네요!
```suggestion
        for (long i = 0; i < 12; i++) {
            bookmarkRepository.save(new Bookmark(bookmarkType, i + 1, memberId));
        }
```